src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    public void doWork() {
+        System.out.println("Bulldozer start work");

```

Comment:
The message 'Bulldozer start work' is not very informative. It would be better to specify what type of work the Bulldozer is doing. For example, 'Bulldozer is clearing the construction site.'